doc/md/installation.md

```diff
@@ -227,6 +227,24 @@ $ sudo dnf install \
     cairo-gobject-devel
 ```
 
+For openSUSE stable (Leap 15.5):
```

Comment:
can you add a note here that pipx is required (I assume it is?)

Reply:
It throws the externally-managed error as described below. Therefor I didn't explicitely write it here (as neither was done for the other distris).

Reply:
Well but in this one you list pipx in the requirements

Reply:
Is it possible to still add this note? @sklemer1